app.js

```diff
@@ -204,3 +204,20 @@ popImg.addEventListener('click', () => {
 closeCross.addEventListener('click', () => {
   wrapPopup.classList.remove('active');
 });
+
+// FORM VALIDATION //
```

Comment:
- [ ] [OPTIONAL] Kindly modify your README file so it looks more professional, you can remove the headers that your not using 
```
Getting Started
To get a local copy up and running follow these simple example steps.

Prerequisites
Setup
Install
Usage
Run tests
Deployment

```